client/reducers/conditionReducer.js

```diff
@@ -1,20 +1,35 @@
-import { handleActions } from 'redux-actions';
-import * as actionTypes from 'actions/conditionActionTypes';
+import CONDITION_ACTIONS from '../actions/conditionActionTypes';
+import { REQUEST, SUCCESS, ERROR } from '../actions/constants';
 
-const initialState = {
+export const initialState = {
   radius: 500,
+  latitude: null,
+  longitude: null,
+};
+/* Selectors */
+export const hasCompleteConditions = (state) => {
+  return state && state.radius && state.latitude && state.longitude;
+};
+
+/* Reducer */
+const conditionReducer = (state = initialState, action) => {
```

Comment:
It's more of a preference I think since I ended up not using createActions with my actions. I preferred the clearer syntax of the switch statement where I don't need to keep writing action and state as arguments and if there are bugs it's more easily traceable since switch statement is a feature of the language